roles/matrix-bot-go-neb/defaults/main.yml

```diff
@@ -0,0 +1,225 @@
+# Go-NEB is a Matrix bot written in Go. It is the successor to Matrix-NEB, the original Matrix bot written in Python.
+# See: https://github.com/matrix-org/go-neb
+
+matrix_bot_go_neb_enabled: false
```

Comment:
We usually default roles to enabling their services (which eases people who are including our roles manually in their own other playbook).

We then disable services in `group_vars/matrix_servers` as needed. Example line to put in `group_vars/matrix_servers`:

```yaml
matrix_bot_go_neb_enabled: false
``

Reply:
Adapted in https://github.com/Zir0h/matrix-docker-ansible-deploy/commit/ce14e3e8af7fbd884c4b7ab1b40b7e662a31548d